<ui version="4.0" >
 <class>contributionvoucherscanvcClass</class>
 <widget class="QWidget" name="contributionvoucherscanvcClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NA21 ContributionVoucherScanVC</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title" >
    <string>Scan overview</string>
   </property>
   <widget class="QListWidget" name="listWidget" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>256</width>
      <height>191</height>
     </rect>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy" >
     <enum>Qt::NoContextMenu</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblVouchersScanned" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Vouchers Scanned: 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title" >
    <string>Scan Area</string>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Scan here</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="scannedContributionVoucherNo" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>30</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClose" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>290</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy" >
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
